transformers4rec/torch/model/prediction_task.py

```diff
@@ -274,7 +274,7 @@ def calculate_metrics(  # type: ignore
 
         outputs = {}
         if forward:
-            predictions = self(predictions)
+            predictions = self(predictions, ignore_masking=False)
```

Comment:
why False?

Reply:
To compute metrics and losses we need to compute the masked targets.